servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/AsyncContext.java

```diff
@@ -133,20 +294,38 @@ public static void putAll(Map<AsyncContextMap.Key<?>, Object> map) {
      * @throws UnsupportedOperationException if this method is not supported by the underlying {@link AsyncContextMap}
      * implementation.
      * @see AsyncContextMap#removeAll(Iterable)
+     * @deprecated Use {@link #removeAllPairs(Iterable)}
      */
-    public static boolean removeAll(Iterable<AsyncContextMap.Key<?>> entries) {
+    @Deprecated
+    public static boolean removeAll(final Iterable<AsyncContextMap.Key<?>> entries) {
```

Comment:
Java complains if I have two methods that take a `Map` of something, because parameters will be replaced at compilation time. Therefore, had to come up with a different name. We can add a `putAll` alias (and deprecate `putAllFromMap`) in 0.42. I will add a comment to clarify.

Reply:
Added an alias in #1959